js/pihole/footer.js

```diff
@@ -8,6 +8,14 @@ $("body").on("click", function(event) {
     }
 });
 
+// Temperature display toogle
```

Comment:
This has to be done **after** loading jQuery, so I put it into footer.js rather than into header.js.